src/Resources/views/CRUD/base_list.html.twig

```diff
@@ -66,7 +66,7 @@ file that was distributed with this source code.
                                             {% if field_description.options.sortable is defined and field_description.options.sortable %}
                                                 {% set sortable             = true %}
                                                 {% set sort_parameters      = admin.modelmanager.sortparameters(field_description, admin.datagrid) %}
-                                                {% set current              = admin.datagrid.values._sort_by == field_description or admin.datagrid.values._sort_by.name == sort_parameters.filter._sort_by %}
+                                                {% set current              = admin.datagrid.values._sort_by is defined and (admin.datagrid.values._sort_by == field_description or admin.datagrid.values._sort_by.name == sort_parameters.filter._sort_by) %}
```

Comment:
Please break this line or move code to PHP

Reply:
@core23 Done, is it enough ?